Fitness/Views/ProRoutineView.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ProRoutineView.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+class ProRoutineView: UIView {
+
+    var routine: ProRoutine!
+    
+    var containerView: UIView!
+    var titleLabel: UILabel!
+    var routineTypeImage: UIImageView!
+    var routineTypeLabel: UILabel!
+    var addButton: UIButton!
+    var stepsTextView: UITextView!
```

Comment:
prefer these to be alphabetized or organized in some manner

Reply:
I'll go through and alphabetize the UI elements for the classes I edited